docs/docs/configurations.md

```diff
@@ -0,0 +1,348 @@
+# All Optional Arguments
+
+Three main configuration classes are specific in `zamba`:
+
+* `VideoLoaderConfig`: Defines all possible parameters for how videos are loaded
+* `PredictConfig`: Defines all possible parameters for model inference
+* `TrainConfig`: Defines all possible parameters for model training
+
+Each inherits from the pydantic [BaseModel](https://pydantic-docs.helpmanual.io/usage/models/) as a parent class.
+
+<a id='video-loading-arguments'></a>
+
+## Video loading arguments
+
+The `VideoLoaderConfig` class <!-- TODO: add link to source code><!--> defines all of the optional parameters that can be specified for how videos are loaded before either inference or training. This includes selecting which frames to use from each video.
+
+```python
+>> from zamba.data.video import VideoLoaderConfig
+>> default_video_loader = VideoLoaderConfig()
+>> default_video_loader
+
```

Comment:
for `PredictConfig` below we use `help(PredictConfig)` instead of instantiating and printing; prefer to do the same for both examples

Reply:
Agreed! I actually think this whole section is mostly repetitive now. I'll just add the full python code for specifying the same defaults as the YAML file for reference at the end of `yaml_config.md`, and get rid of the rest of this section